src/modules/party/dto/post-party-response.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class PostPartyResponse {
+  @ApiProperty({ readOnly: true, required: false })
+  partyIdentifier?: string;
```

Comment:
Why is this optional?

Reply:
In the Mulesoft code, if the first matching party returned by `GET /Party/Search/{searchText}` has an empty `partyIdentifier`, we still return a `200` response, but with a empty object as the body.